cumulus/steps/ec2/role.py

```diff
@@ -0,0 +1,32 @@
+from troposphere import Ref
+
+from cumulus.chain import step
+from troposphere.iam import InstanceProfile
+
+from cumulus.util.tropo import TemplateQuery
+
+
+class Role(step.Step):
```

Comment:
This role is specific to an instance profile, so we might want to name it specifically that.   

There will be many other things that want to call themselves "Role_xxx".